src/SFA.DAS.ApprenticeFeedback.Application.UnitTests/Queries/WhenRequestingFeedbackTargetsForUpdate.cs

```diff
@@ -0,0 +1,437 @@
+{
+    public class WhenRequestingFeedbackTargetsForUpdate
+    {
+        #region Test Apprentice Feedback Targets
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_NotStarted(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = null,
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_Completed(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Complete
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_Withdrawn(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Withdrawn
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_EligibilityCalculatedRecently(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-2),
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_NoFeedback(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-100),
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_FeedbackGivenRecently(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-100),
+                ApprenticeFeedbackResults = new List<Domain.Entities.ApprenticeFeedbackResult>() 
+                { 
+                    new Domain.Entities.ApprenticeFeedbackResult()
+                    {
+                        DateTimeCompleted = DateTime.UtcNow.AddDays(-2)
```

Comment:
Change this to -14, and I think you'll get a test failure due to the wrong configuration being used in the handler.